--- a/arch/arm/dts/imx6ull-14x14-evk.dts
+++ b/arch/arm/dts/imx6ull-14x14-evk.dts
@@ -9,7 +9,7 @@
 #include "imx6ul-14x14-evk-u-boot.dtsi"
 
 / {
-	model = "i.MX6 ULL 14x14 EVK Board";
+	model = "Freescale i.MX6 ULL 14x14 EVK Board";
 	compatible = "fsl,imx6ull-14x14-evk", "fsl,imx6ull";
 };
 
@@ -18,13 +18,3 @@
 			  <&clks IMX6UL_CLK_PLL4_AUDIO_DIV>;
 	assigned-clock-rates = <320000000>, <786432000>;
 };
-
-&csi {
-	status = "okay";
-};
-
-&ov5640 {
-	status = "okay";
-};
-
-/delete-node/ &sim2;
